Docs/update nextjs quickstart (#9613)
supabase/supabase: apps/reference/docs/guides/with-nextjs.mdx

Install the Supabase Auth UI for React

```bash
npm install @supabase/auth-ui-react
```

Add the `Auth` component to your home page

```jsx title="pages/index.js"
import type { NextPage } from 'next'
import { Auth, ThemeSupa } from '@supabase/auth-ui-react'
import { useSession, useSupabaseClient } from '@supabase/auth-helpers-react'

const Home: NextPage = () => {
  const session = useSession()
  const supabase = useSupabaseClient()

  return (
    <div className="container" style={{ padding: '50px 0 100px 0' }}>
      {!session ? (
        <Auth
          supabaseClient={supabase}
          appearance={{ theme: ThemeSupa }}
          theme="dark"
        />
      ) : (
        <p>Account page will go here.</p>
      )}
    </div>
  )
}

export default Home
```

### Account page

After a user is signed in we can allow them to edit their profile details and manage their account.

<Tabs
  groupId="language"
  defaultValue="js"
  values={[
    {label: 'JavaScript', value: 'js'},
    {label: 'TypeScript', value: 'ts'},
  ]}>
<TabItem value="js">

> Let's create a new component for that called `Account.js` within a `components` folder.

Reviewer: ```suggestion
In a `components` folder, create an `Account.js` component:
```

}
```

</TabItem>
<TabItem value="ts">

> Let's create a new component for that called `Account.tsx` within a `components` folder.

Reviewer: ```suggestion
In a `components` folder, create an `Account.tsx` component:
```
